x-pack/legacy/plugins/actions/server/action_type_registry.test.ts

```diff
@@ -31,6 +34,12 @@ const actionTypeRegistryParams = {
 
 beforeEach(() => jest.resetAllMocks());
 
+beforeAll(() => {
```

Comment:
It's not immediately clear that fake timers are required for this particular test - could be, but looks like maybe not; no harm, I assume, even if not needed